sdk/documentintelligence/azure-ai-documentintelligence/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,200 @@
+- `"prebuilt-document"` model is retired, this model is essentially `"prebuilt-layout"` with `DocumentAnalysisFeature.FORMULAS` specified.
+  _(This is only supported as an optional feature for "prebuilt-layout" and "prebuilt-invoice".)_.
+
+  ```java
+  File document = new File("{your-file-to-analyze}");
+  SyncPoller<AnalyzeResultOperation, AnalyzeResultOperation> analyzeLayoutResultPoller =
+          client.beginAnalyzeDocument("prebuilt-layout", null,
+                  null,
+                  null,
+                  Arrays.asList(DocumentAnalysisFeature.FORMULAS),
+                  null,
+                  null,
+                  new AnalyzeDocumentRequest().setBase64Source(Files.readAllBytes(document.toPath())));
+    ```
+
+  For the complete sample, see [Sample: KeyValuePair][Sample-AnalyzeAddOnKeyValuePair.
+
+- All prebuilt models can be seen [here][service_supported_models]. If you wish to still use these models, please rely on the older 
+  `azure-ai-formrecognizer` library through the older service API versions.
+
+## Additional samples
+
+For additional samples please take a look at the [Document Intelligence Samples][README-Samples]
+
+<!-- Links -->
+[DocumentIntelligenceClientBuilder]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/main/java/com/azure/ai/documentintelligence/documentanalysis/DocumentIntelligenceClientBuilder.java
+[DocumentIntelligenceClient]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/main/java/com/azure/ai/documentintelligence/documentanalysis/DocumentIntelligenceClient.java
+[DocumentIntelligenceAsyncClient]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/main/java/com/azure/ai/documentintelligence/documentanalysis/DocumentIntelligenceAsyncClient.java
+[DocumentIntelligenceAdministrationClientBuilder]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/main/java/com/azure/ai/documentintelligence/documentanalysis/administration/DocumentIntelligenceAdministrationClientBuilder.java
+[DocumentIntelligenceAdministrationClient]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/main/java/com/azure/ai/documentintelligence/documentanalysis/administration/DocumentIntelligenceAdministrationClient.java
+[DocumentIntelligenceAdministrationAsyncClient]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/documentintelligence/azure-ai-documentintelligence/src/main/java/com/azure/ai/documentintelligence/documentanalysis/administration/DocumentIntelligenceAdministrationAsyncClient.java
+[FormRecognizerMigrationInstantiateClient]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/migration-guide.md#instantiating-clients
```

Comment:
Looks this one is redundant, I don't see it is used in this file.

Reply:
Yeah. You are right